src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
```

Comment:
The function parameter `days` should be named more descriptively to reflect that it represents the number of days the car is rented. For example, `numberOfDays` would be clearer.